webpack.config.prod.js

```diff
@@ -0,0 +1,38 @@
+const webpack = require('webpack');
+
+module.exports = {
+  entry: "./src/js/main",
+  output: {
+    filename: "build.js",
+  },
+
+  module: {
+    loaders: [
+      {
+        test:   /\.js$/,
+        loader: 'babel'
```

Comment:
where do you set babel configuration?
I do not see any .babelrc file or query parameter in this loader